mne/preprocessing/tests/test_maxwell.py

```diff
@@ -128,11 +136,15 @@ def test_maxwell_filter_additional():
     raw_sss.save(test_outname)
     raw_sss_loaded = Raw(test_outname, preload=True, proj=False,
                          allow_maxshield=True)
+
     # Some numerical imprecision since save uses 'single' fmt
     assert_allclose(raw_sss_loaded._data[:, :], raw_sss._data[:, :],
                     rtol=1e-6, atol=1e-20)
 
-    # Test covariance calculation XXX add this
+    # Test covariance calculation
+    cov_raw = compute_raw_data_covariance(raw)
+    cov_sss = compute_raw_data_covariance(raw_sss)
 
+    assert_equal(np.rank(cov_raw), np.rank(cov_sss))
```

Comment:
and you also need to do it on `cov_raw['data']` or so, you're passing an object to `np.rank` here instead of the data from that object...